COMPILING.md

```diff
@@ -0,0 +1,143 @@
+## Points to note
+
+- Jamulus can be compiled for Linux, Windows and macOS. However, the preferred method of supporting these platforms is to use the binaries generated by the autobuild process in the Jamulus repository.
+- For unattended installs, see the contributed [installation scripts](https://github.com/jamulussoftware/installscripts)
+- There are reports from people who successfully compile and run Jamulus on BSDs.
+- Android and iOS are not officially supported, therefore there are no instructions yet.
+
+---
+
+
+## Download sources
+
+For .tar.gz [use this link](https://github.com/jamulussoftware/jamulus/archive/latest.tar.gz) to download the latest release
+
+For .zip [use this link](https://github.com/jamulussoftware/jamulus/archive/master.zip)
+
+## Linux
+
+#### Install dependencies
+
+On Ubuntu-based distributions 18.04+, Debian 9+ or 10 and Raspberry Pi Raspbian Buster release or later:
+
+* build-essential
+* qt5-qmake
+* qtdeclarative5-dev
+* qt5-default
+* qttools5-dev-tools
+* libjack-jackd2-dev
+
+#### On Fedora
+
+* qt5-qtdeclarative-devel
+* jack-audio-connection-kit-dbus
+* libqt5concurrent5 (qt5-linguist for Fedora 33)
+* jack-audio-connection-kit-devel
+
+#### For all desktop distributions
+
+[QjackCtl](https://qjackctl.sourceforge.io/) is optional, but recommended to configure JACK.
+
+
+### Standard desktop build
+
+```
+make distclean
+qmake ;# qmake-qt5 on Fedora 33
+make
+sudo make install
+```
+
+`make distclean` is optional but ensures a clean build environment. `make install` is optional and puts the Jamulus binary into `/usr/local/bin`.
+
+**Users of Raspberry Pi** You may want to compile the client on another machine and run the binary on the Raspberry Pi. In which case the only libraries you need to run it are those for a [headless server](Server-Linux#running-a-headless-server) build, but _with_ the JACK sound packages. 
+
+
+### “Headless” server build
+
+Although not strictly necessary, we recommend using the headless flag to speed up the build process. Gentoo users may also be able to avoid installing some dependencies as a consequence. Compiling with the headless flag means you can avoid installing some of the dependent packages, save some disk space and/or speed up your build time. 
```

Comment:
Reduction of dependencies and build time speedup are mentioned twice, so this sounds a bit redundant and might be condensed into a single sentence?

```suggestion
Although not strictly necessary, we recommend using the headless flag to avoid having to install some of the dependent packages, save some disk space and/or speed up your build time. 
```